src/components/Checkbox.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <label class="checkbox">
+    <span class="text">{{ text }}</span>
+    <input
+      v-model="model"
+      type="checkbox"
+    >
+    <span class="checkmark" />
+  </label>
+</template>
+
+<script>
+export default {
+  name: 'Checkbox',
+  props: {
+    getter: { type: Boolean, required: true },
+    text: { type: String, default: '' },
+    setter: { type: Function, required: true },
+  },
+  computed: {
+    model: {
+      get() {
+        return this.getter;
```

Comment:
IMHO this should be `return this.getter()`

Reply:
get() is a function that returns a value that corresponds to the checkbox state. Probably you don't like the name of the property "this.getter" - I could change it, but the logic is fine